utils.py

```diff
@@ -336,11 +336,11 @@ def get_app_port_mappings(app):
     portMappings = app.get('container', {})\
                       .get('docker', {})\
                       .get('portMappings')
-    if portMappings:
+    if portMappings is not None:
         return portMappings
 
     portMappings = app.get('container', {})\
-                      .get('portMappings')
+                      .get('portMappings', [])
```

Comment:
we should remove the default and then have a specific check to return `[]` if None. This is because `app` is may have None as the value.

We partially mitigated this elsewhere with the "cleanup_json" function, but there is one other place in the code that gets json (marathon.list()) which we don't sanitize

Reply:
we can either try to sanitize all entrypoints of json (like marathon.list()) or have this function also handle a `None` aka json null

Reply:
Please check https://github.com/mesosphere/marathon-lb/pull/496 before we move forward with this PR